docs/configuration.md

```diff
@@ -214,6 +214,19 @@ services.bitcoind = {
 };
 ```
 
+> Note: If a `secure-node.nix` or `tor-enable.nix` preset is imported in your
+> configuration or a `tor.enforce` option is explicitly enabled, you also need
+> to allow remote connections for **every** service which needs to connect to
+> the remote bitcoind:
+> 
+> ```
+> systemd.services.<service>.serviceConfig = {
+>   IPAddressAllow = ["10.10.0.2/32"];
+>   IPAddressAllow = [ ${services.bitcoind.rpc.address} ];
+> };
+> ```
```

Comment:
Is there a particular reason that this is a blockquote? I don't think we do this anywhere else.